project/Build.scala

```diff
@@ -40,7 +40,7 @@ object ScaldingBuild extends Build {
   val scalameterVersion = "0.6"
   val scroogeVersion = "3.17.0"
   val slf4jVersion = "1.6.6"
-  val thriftVersion = "0.5.0"
+  val thriftVersion = "0.7.0"
```

Comment:
I was seeing failed builds due to 404's -- looks like the 0.5.0 version is no longer listed on maven central.. too old i guess